pkg/auth/handler/webapp/settings.go

```diff
@@ -26,19 +31,72 @@ func ConfigureSettingsRoute(route httproute.Route) httproute.Route {
 		WithPathPattern("/settings")
 }
 
+type SettingsViewModel struct {
+	Authenticators           []*authenticator.Info
+	RecoveryCodes            []*mfa.RecoveryCode
+	SecondaryTOTPEnabled     bool
```

Comment:
It seems there are used to determine whether the relevant sections should be shown. However, if user has a secondary authenticator not enabled in config (i.e. config changed), should we still show that section?